inventory-items.go

```diff
@@ -0,0 +1,40 @@
+package shopify
+
+// InventoryItemRepository manages inventory items
+type InventoryItemRepository interface {
+	// Get retrieves an inventory item by its id
+	Get(id int64) (InventoryItem, error)
+}
+
+type CountryHarmonizedSystemCode struct {
```

Comment:
Think `CountryHarmonizedSystemCode` should have some description since is an exported type

Reply:
Ah it doesn't need to be exported, good shout 🙂